app/controllers/books_controller.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class BooksController < ApplicationController
+  before_action :set_book, only: %i[show edit update destroy]
+
+  def index
+    @books = Book.page params[:page]
+  end
+
+  def new
+    @book = Book.new
+  end
+
+  def show; end
+
+  def edit; end
+
+  def create
```

Comment:
take care of authors: you have already defined models `author` and `book_author`, there is nothing to split